### PR TITLE
fix: ensure the correct type when signing a message using eth-account

### PR DESCRIPTION
**Pull request by fubuloubu**
### What I did
Noticed that when you sign a message using eth-account, it does not return `r` and `s` in the signature the same as with signing a transaction

### CODE REVIEW

**Review by unparalleled-js — APPROVED** [review]
Looks good! I reviewed on my phone. Is there any way we can test this?

**Comment by fubuloubu** [comment]
> Looks good! I reviewed on my phone. Is there any way we can test this?

we ran into this bug with `ape-flashbots` when we had signed a message and tried to convert it to a string and got a failure because `r` and `s` were `int` instead of `bytes`

could probably unit test thing fairly easily :eyes:

File: src/ape_accounts/accounts.py [hunk 1]
```diff
@@ -131,11 +132,19 @@ def sign_message(self, msg: SignableMessage) -> Optional[MessageSignature]:
             return None
 
         signed_msg = EthAccount.sign_message(msg, self.__key)
-        return MessageSignature(v=signed_msg.v, r=signed_msg.r, s=signed_msg.s)  # type: ignore
+        return MessageSignature(  # type: ignore
+            v=signed_msg.v,
```
Comment (unparalleled-js) [inline src/ape_accounts/accounts.py:136]: Kind of weird, I just noticed we do the opposite of this in `ape.types.signatures`

Comment (fubuloubu) [reply src/ape_accounts/accounts.py:136]: I think what you see is that `v` is converted to `bytes`

this logic is somewhat messed up, yeah

**Comment by unparalleled-js** [comment]
I did some work on `ape-keyring` over this break and I ran into this issue there as well.

**Comment by fubuloubu** [comment]
> I did some work on `ape-keyring` over this break and I ran into this issue there as well.

Let's definitely work on this Monday. It's a blocker

**Comment by unparalleled-js** [comment]
For consistency, we should fix this in the `TestAccount` (`ape_test.accounts.py`) as well.
That returns `int` instead of bytes here

**Comment by fubuloubu** [comment]
> For consistency, we should fix this in the `TestAccount` (`ape_test.accounts.py`) as well. That returns `int` instead of bytes here

fixed in b0dc7ce

**Review by unparalleled-js — APPROVED** [review]

**Pull request merged**